docs/manual/manual_redundant_throttle.rst

```diff
@@ -0,0 +1,113 @@
+
+.. note:: 
+    Due to Vertiq modules' hardware, it is not possible to use both IQUART and hobby protocols simultaneously. As such, it is only possible to use DroneCAN with one of IQUART 
+    or hobby as redundant throttle sources.
+
+Configuration Example 1
+##########################
+
+Suppose your module supports DroneCAN, IQUART, and hobby inputs. 
+
+You configure the following:
+
+- IQUART's priority to 3 and DroneCAN's to 1 (the module will automatically set hobby's to 2 ensuring that there are no two matching priorities)
+- Flight controller outputting both :ref:`IQUART Flight Controller Interface commands <controlling_ifci>` as well as DroneCAN commands
+- Throttle source timeout configured to 0.5 seconds
+
+The sequence diagram below illustrates how your module reacts to received throttle commands as well as how it deals with switching between sources.
+
+.. image:: ../_static/manual_images/redundant_throttle/iquart_prio_example.png
+    :width: 50%
+
+Note that the first DroneCAN throttle is applied to the motor since no IQUART messages had been received before. Then, when both an IQUART and DroneCAN message are received, 
+the IQUART throttle is applied. Last, DroneCAN throttles are only applied again once the throttle timeout is reached after IQUART disconnects. If for any reason, the module begins receiving 
+IQUART throttles again at this point, the module will immediately start accepting those as IQUART has a higher priority.
+
+Configuration Example 2
+##########################
+
+Suppose your module supports DroneCAN, IQUART, and hobby inputs. 
+
+You configure the following:
+
+- DroneCAN's priority to 3, IQUART to 0, hobby to 0
+- Flight controller outputting both :ref:`hobby_dshot` as well as DroneCAN commands
+- Throttle source timeout configured to 0.1s
+
+.. image:: ../_static/manual_images/redundant_throttle/dshot_ignored_ex.png
+    :width: 70%
+
+In this example, the module is configured to ignore all hobby and IQUART throttle messages. So, all received DSHOT throttles are dropped, and throttles are only applied once DroneCAN 
+throttles are received. An important note from this example is that the module will reach its :ref:`propeller motor control timeout <manual_timeout>` even if the 
+flight controller continues to send DSHOT throttles. As they are not processed by the throttle source manager, DSHOT throttles cannot be used to reset the timeout timer.
+
+********************************
+Redundant Throttle and Arming
+********************************
+
+All throttle commands received by the throttle source manager are sent to your module's :ref:`arming handler <manual_advanced_arming>`, and are subject to all constraints defined by the arming handler. 
+For example, suppose you have configured IQUART to be the highest priority source, and are sending IQUART throttle commands via the :ref:`Vertiq Testing Tool <vertiq_testing__guide>`. At the same 

```

Comment:
Should add a (VTT) after Vertiq Testing Tool, to define the acronym if you are going to use that acronym later on